api/filters.go

```diff
@@ -124,6 +124,118 @@ func (api *API) createFilter(w http.ResponseWriter, r *http.Request) {
 	api.respond.JSON(ctx, w, http.StatusCreated, resp)
 }
 
+func (api *API) submitFilter(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	filterID := chi.URLParam(r, "id")
+	logData := log.Data{
+		"filter_id": filterID,
+	}
+
+	filter, err := api.store.GetFilter(ctx, filterID)
+	if err != nil {
+		// error 500
+		api.respond.Error(
+			ctx,
+			w,
+			//			statusCode(err),
+			500,
```

Comment:
I would use the httpStatus 500 constant here.

Reply:
used constants